src/genotyper.cc

```diff
@@ -737,6 +769,12 @@ Status genotype_site(const genotyper_config& cfg, MetadataCache& cache, BCFData&
         losses.insert(make_pair(sample_name,loss));
     }
     merge_loss_stats(losses, losses_for_site);
+
+    if (residuals != nullptr && any_losses(losses)) {
```

Comment:
Initially, I also thought these two conditions were equivalent. I am not sure exactly why, but they are not. There are many fewer cases with "any_losses". I think this is because sometimes the originals are also missing calls.